src/browser/extension/options/index.js

```diff
@@ -1,79 +1,23 @@
 import React from 'react';
 import { render } from 'react-dom';
+import Options from './options';
 
 chrome.runtime.getBackgroundPage(background => {
   const syncOptions = background.syncOptions;
 
-  const saveOption = e => {
-    let value;
-    if (e.target.type === 'checkbox') value = e.target.checked;
-    else if (
-      e.target.type === 'input' || e.target.type === 'text'
-    ) value = Number(e.target.value);
-    else value = e.target.value.trim();
-    syncOptions.save(e.target.id, value);
+  const saveOption = (name, value) => {
+    syncOptions.save(name, value);
   };
 
-  const isValidRegex = str => {
```

Comment:
I also removed the `isValidRegex` and `saveUrls` functions as a part of [moving `saveOption`](https://github.com/zalmoxisus/redux-devtools-extension/pull/54/files#r70189641). They don’t actually do anything specific, and removing them helped me to move `saveOption` logics to the new place.